python/tools/dht/tests.py

```diff
@@ -249,21 +249,22 @@ def _reset(self):
     @staticmethod
     def lookupCb(vals, prefix):
         PhtTest.indexEntries = list(vals)
-        PhtTest.prefix = prefix.decode()
-        DhtNetwork.log('Index name: <todo>')
-        DhtNetwork.log('Leaf prefix:', prefix)
+        PhtTest.prefix = str(prefix)
+        # TODO: output index name
+        # DhtNetwork.Log.log('Index name: <todo>')
+        DhtNetwork.Log.log('Leaf prefix:', PhtTest.prefix)
         for v in vals:
-            DhtNetwork.log('[ENTRY]:', v)
+            DhtNetwork.Log.log('[ENTRY]:', v)
 
     @staticmethod
     def lookupDoneCb(ok):
-        DhtNetwork.log('[LOOKUP]:', PhtTest.key, "--", "success!" if ok else "Fail...")
+        DhtNetwork.Log.log('[LOOKUP]:', PhtTest.key, "--", "success!" if ok else "Fail...")
```

Comment:
@kaldoran: What do you mean precisely? `Dhtnetwork.Log` is a class which has methods `log`, `warn` and `err`.

Do you suggest changing `Log.log` method to `Log.debug`?